modules/os2forms_organisation_openid_connect/os2forms_organisation_openid_connect.info.yml

```diff
@@ -0,0 +1,5 @@
+name: OS2Forms Organisation OpenID Connect
+type: module
+description: Module for OS2Forms Organisation OpenID Connect.
+package: OS2Forms
+core_version_requirement: ^9
```

Comment:
This module should probably depend on the OS2Forms Organisation module.

Reply:
Seems reasonable. Added dependency.